src/user-icon/user-icon.scss

```diff
@@ -67,39 +57,38 @@
     z-index: 1;
     left: 0;
     top: 0;
-    width: $size;
-    height: $size;
+    width: 100%;
+    height: 100%;
     border-width: 0;
     border-radius: 50%;
   }
 
-  > .iui-user-icons-status {
+  > .iui-status {
     display: block;
     position: absolute;
     right: -2px;
     bottom: -2px;
-    width: $size / 4;
-    height: $size / 4;
+    width: calc(100% / 4);
+    height: calc(100% / 4);
     border-radius: 50%;
     z-index: 3;
-
     @include themed {
       border: 2px solid t(iui-color-background-1);
       background-color: t(iui-color-background-1);
       box-shadow: inset 0 0 0 1px
         rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-3));
     }
 
-    > .iui-user-icons-status-symbol {
+    > .iui-status-symbol {
       @include themed {
         fill: rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-3));
       }
-      width: $size / 6;
-      height: $size / 6;
+      width: calc(100% / 1.5);
+      height: calc(100% / 1.5);
       display: block;
       position: absolute;
-      right: $size / 24;
-      bottom: $size / 24;
+      right: calc(100% / 6);
+      bottom: calc(100% / 6);
```

Comment:
where do these calculations come from?

Reply:
The `calc(100% / 6)` have been removed, the icons are now centered using flex instead.

Reply:
there is still `(100% / 1.5)` for the size but I guess that one kinda makes sense because it's making the status symbol 2/3 of the container. Idk